agent/bench-scripts/pbench-run-benchmark.pl

```diff
@@ -59,6 +59,8 @@
     exit 1;
 }
 
+print "WARNING: pbench-run-benchmark is deprecated and will be removed in the next release.\n";
```

Comment:
You don't want to offer guidance on a replacement path?

Reply:
I don't know what guidance to offer at this point.  Ideally we'd have a path via https://github.com/cloud-bulldozer/benchmark-wrapper, but we are not there yet.

For this PR I think it is enough to make changes to issue the `WARNING`, and then work via the release notes and other documentation later when a path becomes available.

Reply:
Removing this doesn't reduce our capability for the user, right?  All they have to do is run their choice of benchmark directly, right?  So, a little suggestion here would go a long way.

Reply:
Well, sort of, but not really.  We have `pbench-user-benchmark`, but `pbench-run-benchmark` is a "generalized" `pbench-fio` combined with `pbench-uperf`.  It has the ability to run `fio`, `uperf`, `trafficgen`, etc. but from one script.  We really don't have an alternative for that, other than the existing scripts.

Reply:
Right, but if someone was using `pbench-run-benchmark` to run fio, they can recover from the deprecation by running `pbench-fio`, right?  (And, yeah, for the "gap" cases, they can use `pbench-user-benchmark`.)  So, it would be helpful for the message to point that out.

(If you really think that no one is using `pbench-run-benchmark`, then obviously there's little point to enhancing this, but if someone uses this to run a benchmark that we have support for in another script, it seems like we could point them to that script, and otherwise we could suggest `pbench-user-benchmark`.)

Reply:
And the `0.69` release doesn't even work.  So maybe we should just get rid of it entirely:

```
# pbench-run-benchmark
Empty compile time value given to use lib at /opt/pbench-agent/bench-scripts/pbench-run-benchmark line 19.
Use of uninitialized value in require at /opt/pbench-agent/bench-scripts/pbench-run-benchmark line 23.
Can't locate PbenchCDM.pm in @INC (@INC contains:  /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at /opt/pbench-agent/bench-scripts/pbench-run-benchmark line 23.
BEGIN failed--compilation aborted at /opt/pbench-agent/bench-scripts/pbench-run-benchmark line 23.
```

Reply:
Let's just merge this commit as is, and I'll take that removal up in another commit.

Reply:
👍